### PR TITLE
Core/Spell: Paladin(Blinding Light).

### PR DESCRIPTION
**Pull request by ghost**
**Changes proposed**:
- Paladin(Blinding Light).

**Target branch(es)**: 6x

**Issues addressed**: Fixes # Emits a dazzling light in all directions, blinding enemies within 10 yards, causing them to wander disoriented for 6 sec.

**Tests performed**: (Does it build, tested in-game, etc)
- Compile Without Errors
- Bosses Tested In-Game

**Additional comments**:
1. Need Feedback.
2. Here is a [video](https://youtu.be/EottZrm_HBs) work spells.

### CODE REVIEW

File: sql/custom/world/.gitignore [hunk 1]
```diff
@@ -1 +0,0 @@
-*.sql
```
Comment (Vincent-Michael) [inline sql/custom/world/.gitignore]: why?

File: sql/custom/world/2016_03_19_00_qc_world.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+-- Blinding Light
```
Comment (Vincent-Michael) [inline sql/custom/world/2016_03_19_00_qc_world.sql]: why custom?

Comment (DevQuad) [inline sql/custom/world/2016_03_19_00_qc_world.sql]: Because I 'm doing this for my repository.

Comment (DDuarte) [inline sql/custom/world/2016_03_19_00_qc_world.sql]: Since the intention is to include it in the TC repo, the sql should be added to /sql/updates/world

**Pull request closed without merging**